File: src/main/java/com/paragon/client/systems/module/IgnoredByNotifications.java
package com.paragon.client.systems.module;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * This annotation is used to mark a module to be ignored by the notification system.
 * Examples: ClickGUI, HUDEditor
 */
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.TYPE)
public @interface IgnoredByNotifications {
}

File: src/main/kotlin/com/paragon/impl/module/annotation/IgnoredByNotifications.kt
package com.paragon.impl.module.annotation

/**
 * This annotation is used to mark a module to be ignored by the notification system.
 * Examples: ClickGUI, HUDEditor
 *
 * @author Surge
 */
@Target(AnnotationTarget.CLASS)
annotation class IgnoredByNotifications